src/components/menu/API.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from "react";
+import {Trans, withTranslation, WithTranslation} from "react-i18next";
+import {IAlert} from "../../types";
+
+
+interface IState {
+    alerts: IAlert[];
+}
+
+class API extends React.Component<WithTranslation, IState> {
+
+    state = {alerts: []};
```

Comment:
What are those for?

Reply:
hm... i tried to delete everything "alert" but then it would throw errors so i kept it in

Reply:
I would think you can 
- remove the import of IAlert at the top
- remove the whole declaration of the interface IState
- replace `IState` in the class declaration with `null` (?)
- remove the declaration of `state` here

Reply:
ok i managed to get it. IState is object